source/desktop/sharedexchange/sharedexdns.md

```diff
@@ -33,10 +36,10 @@ Please allow time for any DNS changes to propagate through the internet, this ma
 
 * If you have any further questions please contact your dedicated account manager or speak with one of our support engineers on 0800 230 0032.
 
-```eval_rst
+``` eval_rst
```

Comment:
need to remove this space

Reply:
Resolved in commits: 5037700222dc26f675b4c76759c43fbb47664862 & c82fbaa9501eecb13c9573fb3c466b4beb1dda18